smart_open/tests/test_smart_open.py

```diff
@@ -672,168 +615,6 @@ def write_callback(request):
         assert responses.calls[3].request.url == "http://127.0.0.1:8440/file"
 
 
-class S3IterBucketTest(unittest.TestCase):
```

Comment:
These tests have no more sense, am I right?

Reply:
Yes. Most of them moved to the s3 module.